<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>convWindow</class>
 <widget class="QDockWidget" name="convWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>160</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>262</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>262</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Convergência e Declinação</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>56</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Convergência</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>96</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Declinação</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="convEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="declEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Ativar</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
